paper.bib

```diff
@@ -178,47 +178,16 @@ @ARTICLE{Smith2021-mh
   year     =  2021
 }
 
-@ARTICLE{Cohen-Adad2021-tn,
-  title    = "Open-access quantitative {MRI} data of the spinal cord and
-              reproducibility across participants, sites and manufacturers",
-  author   = "Cohen-Adad, Julien and Alonso-Ortiz, Eva and Abramovic, Mihael
-              and Arneitz, Carina and Atcheson, Nicole and Barlow, Laura and
-              Barry, Robert L and Barth, Markus and Battiston, Marco and
-              B{\"u}chel, Christian and Budde, Matthew and Callot, Virginie and
-              Combes, Anna J E and De Leener, Benjamin and Descoteaux, Maxime
-              and de Sousa, Paulo Loureiro and Dost{\'a}l, Marek and Doyon,
-              Julien and Dvorak, Adam and Eippert, Falk and Epperson, Karla R
-              and Epperson, Kevin S and Freund, Patrick and Finsterbusch,
-              J{\"u}rgen and Foias, Alexandru and Fratini, Michela and
-              Fukunaga, Issei and Gandini Wheeler-Kingshott, Claudia A M and
-              Germani, Giancarlo and Gilbert, Guillaume and Giove, Federico and
-              Gros, Charley and Grussu, Francesco and Hagiwara, Akifumi and
-              Henry, Pierre-Gilles and Hor{\'a}k, Tom{\'a}{\v s} and Hori,
-              Masaaki and Joers, James and Kamiya, Kouhei and Karbasforoushan,
-              Haleh and Ke{\v r}kovsk{\'y}, Milo{\v s} and Khatibi, Ali and
-              Kim, Joo-Won and Kinany, Nawal and Kitzler, Hagen H and Kolind,
-              Shannon and Kong, Yazhuo and Kudli{\v c}ka, Petr and Kuntke, Paul
-              and Kurniawan, Nyoman D and Kusmia, Slawomir and Labounek,
-              Ren{\'e} and Lagan{\`a}, Maria Marcella and Laule, Cornelia and
-              Law, Christine S and Lenglet, Christophe and Leutritz, Tobias and
-              Liu, Yaou and Llufriu, Sara and Mackey, Sean and Martinez-Heras,
-              Eloy and Mattera, Loan and Nestrasil, Igor and O'Grady, Kristin P
-              and Papinutto, Nico and Papp, Daniel and Pareto, Deborah and
-              Parrish, Todd B and Pichiecchio, Anna and Prados, Ferran and
-              Rovira, {\`A}lex and Ruitenberg, Marc J and Samson, Rebecca S and
-              Savini, Giovanni and Seif, Maryam and Seifert, Alan C and Smith,
-              Alex K and Smith, Seth A and Smith, Zachary A and Solana,
-              Elisabeth and Suzuki, Y and Tackley, George and Tinnermann,
-              Alexandra and Valo{\v s}ek, Jan and Van De Ville, Dimitri and
-              Yiannakas, Marios C and Weber, II, Kenneth A and Weiskopf,
-              Nikolaus and Wise, Richard G and Wyss, Patrik O and Xu, Junqian",
-  journal  = "Scientific Data",
-  volume   =  8,
-  number   =  1,
-  doi = {10.1038/s41597-021-01044-0},
-  pages    = "219",
-  month    =  dec,
-  year     =  2021,
+@article{Cohen2021-tn,
```

Comment:
Arent you mixing `10.1038/s41597-021-01044-0` and `10.1038/s41596-021-00588-0` into a single ref by this?

Reply:
Are not they different versions of the same article?

Reply:
No!

`10.1038/s41596-021-00588-0` is:
Cohen-Adad, J., Alonso-Ortiz, E., Abramovic, M. et al. Generic acquisition protocol for quantitative MRI of the spinal cord. Nat Protoc 16, 4611–4632 (2021). https://doi.org/10.1038/s41596-021-00588-0

while `10.1038/s41597-021-01044-0` is:
Cohen-Adad, J., Alonso-Ortiz, E., Abramovic, M. et al. Open-access quantitative MRI data of the spinal cord and reproducibility across participants, sites and manufacturers. Sci Data 8, 219 (2021). https://doi.org/10.1038/s41597-021-00941-8

And actually, `10.1038/s41597-021-01044-0` is a correction of `10.1038/s41597-021-00941-8`